.github/ISSUE_TEMPLATE/Mobile_apps_bug_report.md

```diff
@@ -0,0 +1,34 @@
+---
+name: Bug report for Mobile Apps
+about: Create a report to help us improve the Gutenberg mobile apps version
+labels: Mobile App Android/iOS
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Tap on '....'
+3. Scroll down to '....'
+4. See error

```

Comment:
Nit, but let's put something like "See '...exact error...'" to clearly make it look like a placeholder for expanding what the error looks like.